geo_features.js

```diff
@@ -154,11 +158,22 @@ function getFeatureLabel(feature) {
 }
 
 function getFeatureFont(feature) {
-  // leave this useless switch statement here to show the fonts
-  // that have been explored
+  // default styles
   let fontchoice = 9;
   let fontname = 'sans serif';
   let fontscale = 1;
+  let fontcolor = "black";
+
+  // lookup table for entity types and names, mapping to font and font color and other info
+  let entity1name = feature.properties.entity1name;
+  let entity2type = feature.properties.entity2type;
+  let entity2name = feature.properties.entity2name;
+
+  if(entity1name == 'Native American' && entity2type == 'people') {
+      fontchoice   = 0;
```

Comment:
I'd say 'meh' on Rubik, but that's my opinion. Of the 10 "chosen" fonts in the list, I prefer `3` (New Tegomin), but probably there is an even better choice out there. I searched for 'indigenous fonts' but they are mostly too kitschy and/or Native American centric.

Reply:
I didn't look at this in detail. I'll change the font to 3 for now and then explore an improved font later.